CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ CHANGELOG
 
 ### Bots
 - `collectors.mail.collector_mail_url`: Fix bug which prevented marking emails seen due to disconnects from server (#852).
+- `parsers.spamhaus.parser_cert`: Handle/ignore 'AS?' in feed (#1111)
+
+### Packaging
+- Support building for more distributions
+- Use LSB-paths (/etc/intelmq/, /var/lib/intelmq/, /run/intelmq/) (#470)
```

Comment:
applies to all systems or just Debian? (It's LSB, so I would assume it's for all)

Reply:
For all who use the packages. For others, they use their methods (like pip etc), it does not apply.

probably like this?
```
### Packaging
- dep, rpm:
  - use LSB-paths...
```